app/roles/openshift-installer/templates/98-cache-disk.yaml.j2

```diff
@@ -0,0 +1,57 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: 98-cache-disk
+spec:
+  config:
+    ignition:
+      version: 3.1.0
+    storage:
+      disks:
+      - device: /dev/{{ openshift_installer_cache_disk }}
```

Comment:
I would intuitively set `openshift_installer_cache_disk` to /dev/sdb (or whatever disk) rather than sdb - so the full path to the disk...
That's why I wouldn't prepend /dev in this template

Reply:
I almost did that, but there is another config option that asks for the disk to install the OS on. That option doesn't require a /dev prefix because the CoreOS kernel args don't allow the /dev prefix. 

So I wanted to stay consistent with that.